_beats/libbeat/scripts/unpack_dashboards.py

```diff
@@ -6,6 +6,8 @@
 
 def transform_data(data, method):
     for obj in data["objects"]:
+        if "attributes" not in obj:
+            continue
```

Comment:
This change will be overwritten with the next `make update-beats`. Are you putting up a PR in beats with it? Otherwise we should ensure to adapt the `update-beats` script to either not overwrite this file or print out a warning if it changed and would be overwritten.

Reply:
yes, that is what this was trying to say:

> Will look to contribute the beats changes back to beats.

Reply:
filed here: https://github.com/elastic/beats/pull/10701